gencrud/generators/angular.py

```diff
@@ -463,11 +534,11 @@ def generateAngular( config: TemplateConfiguration, templates: list ):
     logger.info( "appModule: {}".format( json.dumps( appModule, indent = 4 ) ) )
     updateAngularAppModuleTs( config, appModule, exportsModules )
 
-    os.remove( os.path.join( config.source.angular, 'app.module.json' ) )
+    os.remove( os.path.join( config.angular.sourceFolder, 'app.module.json' ) )
     copyAngularCommon( config, os.path.abspath( os.path.join( os.path.dirname( __file__ ),
                                                       '..',
                                                       'common-ts' ) ),
-                       os.path.join( config.source.angular, 'common' ) )
+                       os.path.join( config.angular.sourceFolder, 'common' ) )
```

Comment:
so, the configuration file semantics changed and instead of 
source:
    angular: ...
    
   there will be
   
angular:
       sourceFolder: ....
?

Reply:
The gencrud/schema.py schema file does not contain any "sourceFolder" field description. Instead, the classical schema with source: angular: /path  is assumed. So, config.angular.sourceFolder should be a None object?